controllers/news-controllers.js

```diff
@@ -28,8 +28,23 @@ const getArticlesById = (req, res, next) => {
     })
 }
 
+const getCommentsById = (req, res, next) => {
+    const {article_id} = req.params;
+    checkCommentExists(article_id)
```

Comment:
This is a good approach, but `checkCommentExists` should be a util function. Create a utils module, or directory at root or controler directory level and import from there.